exporter/otlpexporter/internal/arrow/exporter_test.go

```diff
@@ -194,13 +212,14 @@ func TestArrowExporterDowngrade(t *testing.T) {
 	bg := context.Background()
 	require.NoError(t, tc.exporter.Start(bg))
 
-	stream, err := tc.exporter.GetStream(bg)
-	require.Nil(t, stream)
+	sent, err := tc.exporter.SendAndWait(bg, twoTraces)
+	require.False(t, sent)
 	require.NoError(t, err)
 
-	// TODO: test the logger was used to report "downgrading"
-
 	require.NoError(t, tc.exporter.Shutdown(bg))
+
+	require.Equal(t, tc.observedLogs.All()[0].Message, "arrow is not supported")
+	require.Contains(t, tc.observedLogs.All()[1].Message, "downgrading")
```

Comment:
just for safety, should we also check the length of this list before we access it?

Reply:
Fixed (in three places).